chart/kubeapps/values.yaml

```diff
@@ -92,6 +92,7 @@ frontend:
   service:
     port: 80
     type: ClusterIP
+    # annotations: {}
```

Comment:
Why commenting it out? if it is empty, it doesn't need to be commented out. And the rest of optional values (like tolerations) are not commented out. Is this a special case?

Reply:
Thanks for catching the inconsistency, no, it is not a special case, I think I just left it like that by mistake because I was testing the differences between `range` used upstream https://github.com/helm/charts/blob/master/stable/wordpress/templates/ingress.yaml#L16 and `with values` that we use in this chart. 

My bad, sorry.

Reply:
Actually, it seems that it is consistent with the way we use annotations in the ingress. 

https://github.com/kubeapps/kubeapps/blob/1af290624f2fb6e0ca1d5972a665d2d389d69888/chart/kubeapps/values.yaml#L143-L145

Reply:
Actually commented out vs empty object in the values.yaml file does not seem to make any difference, so I am wondering why we commented it out in the ingress in the first place Maybe @prydonius could shed some light? 

With the current commented out value, no annotation added

```bash
helm template . -x templates/kubeapps-frontend-service.yaml 
---
# Source: kubeapps/templates/kubeapps-frontend-service.yaml
apiVersion: v1
kind: Service
metadata:
  name: RELEASE-NAME-kubeapps
  labels:
    app: kubeapps
    chart: kubeapps-0.4.0
    release: RELEASE-NAME
    heritage: Tiller
spec:
```
Using the empty object approach instead, no annotation added either

```bash
git diff
diff --git a/chart/kubeapps/values.yaml b/chart/kubeapps/values.yaml
index fa97086c..7b65f76c 100644
--- a/chart/kubeapps/values.yaml
+++ b/chart/kubeapps/values.yaml
@@ -92,7 +92,7 @@ frontend:
   service:
     port: 80
     type: ClusterIP
-    # annotations: {}
+    annotations: {}
   livenessProbe:

 helm template . -x templates/kubeapps-frontend-service.yaml 
---
# Source: kubeapps/templates/kubeapps-frontend-service.yaml
apiVersion: v1
kind: Service
metadata:
  name: RELEASE-NAME-kubeapps
  labels:
    app: kubeapps
    chart: kubeapps-0.4.0
    release: RELEASE-NAME
    heritage: Tiller
spec:
```

Reply:
In case that we decide to change it, I went ahead and created a patch for it https://github.com/kubeapps/kubeapps/pull/599